### DISCUSSION TITLE
Is there a way of get values of a nodes list at once?

### DISCUSSION THREAD

**Discussion by MatheusDemoner** [FreeOpcUa/opcua-asyncio#1199] [Q&A] [answered]
Hi, i'm new in OPC-UA and need to get a list of nodes from a server.

Currently i'm using the node.get_value() method with a async for loop. But my question is if has a way of getting all the values at once.

Thanks, and congratulations for the amazing lib.

**Comment by AndreasHeine** [comment, marked as answer]
sure!

https://github.com/FreeOpcUa/opcua-asyncio/blob/2fcbbe9d18f2129d1c0d9ac51bf2c81b8b2b251b/asyncua/client/client.py#L803

**Reply by AndreasHeine** [reply]
```python
results = await client.read_values(nodes)
```

and some other ways here:
https://github.com/AndreasHeine/opcua-tutorial/blob/main/client/read.py

**Reply by MatheusDemoner** [reply]
Thank you a lot!!!
